query/src/sql/planner/binder/scalar.rs

```diff
@@ -48,6 +52,74 @@ impl ScalarBinder {
+                    true => {
+                        // Function is aggregate function
+                        let mut data_values = Vec::with_capacity(params.len());
+                        for param in params.iter() {
+                            data_values.push(match param {
+                                Literal::Number(val) => DataValue::try_from_literal(val, None)?,
+                                Literal::String(val) => DataValue::String(val.clone().into_bytes()),
+                                Literal::Boolean(val) => DataValue::Boolean(*val),
+                                Literal::Null => DataValue::Null,
+                            })
+                        }
+
+                        let scalar_binder = ScalarBinder::new();
+                        let mut scalar_exprs = Vec::with_capacity(args.len());
+                        for arg in args.iter() {
+                            scalar_exprs.push(
+                                scalar_binder
+                                    .bind_expr(arg, bind_context)?
+                                    .as_any()
+                                    .downcast_ref::<Scalar>()
+                                    .ok_or_else(|| {
+                                        ErrorCode::UnImplement("Can't downcast to Scalar")
+                                    })?
+                                    .clone(),
+                            );
+                        }
+
+                        let col_pairs = bind_context.result_columns();
+                        let mut col_bindings = Vec::with_capacity(col_pairs.len());
+                        for col_pair in col_pairs.into_iter() {
+                            col_bindings.push(bind_context.resolve_column(None, col_pair.1)?);
+                        }
+
+                        let mut fields = Vec::with_capacity(col_bindings.len());
+                        for col_binding in col_bindings.iter() {
+                            fields.push(DataField::new(
+                                col_binding.column_name.as_str(),
+                                col_binding.data_type.clone(),
+                            ))
+                        }
+
+                        let agg_func_ref = AggregateFunctionFactory::instance().get(
+                            name.clone(),
+                            data_values.clone(),
+                            fields,
+                        )?;
+
+                        Ok(Arc::new(Scalar::AggregateFunction {
+                            func_name: name.clone(),
+                            distinct: *distinct,
+                            params: data_values,
+                            args: scalar_exprs,
+                            data_type: agg_func_ref.return_type()?,
+                            nullable: false,
+                        }))
+                    }
+                    false => Err(ErrorCode::UnImplement(format!(
```

Comment:
What's happen with `SELECT sleep(1);`?

Reply:
The case will fail in the parser phase:
```
  --> SQL:1:16
  |
1 | SELECT sleep(1)
  | ------         ^ expected token <FROM>
  | |
  | while parsing `SELECT ...`

```
cc @andylokandy